EOL repos removal: Updates throughout all product docs (#2959)
sensu/sensu-docs: content/sensu-core/1.9/faq.md

> Is Sensu available for Microsoft Windows?

_IMPORTANT: [Sensu Core reached end-of-life (EOL) on December 31, 2019][7], and we [permanently removed][2] the Sensu EOL repository on February 1, 2021. This means the Sensu Core packages are no longer available. To migrate to Sensu Go, read the [Sensu Core migration guide][6]._

> 

Author: ```suggestion

```

> Is Sensu available for Microsoft Windows?

> _IMPORTANT: We [permanently removed][1] the Sensu end-of-life (EOL) repository on February 1, 2021. [Start your migration to Sensu Go][6]._

Author: ```suggestion
_IMPORTANT: [Sensu Core reached end-of-life (EOL) on December 31, 2019][7], and we [permanently removed][2] the Sensu EOL repository on February 1, 2021. This means the Sensu Core packages are no longer available. To migrate to Sensu Go, read the [Sensu Core migration guide][6]._

```

_NOTE: For `config.json`, it is not necessary to have this file present on either a Sensu client or server, provided that you have the rest of the configuration files present._

[1]: https://discourse.sensu.io/t/updated-eol-timeline-for-sensu-core-and-sensu-enterprise-repos/2396
[2]: https://www.amqp.org/
[3]: https://www.rabbitmq.com/which-erlang.html
[4]: ../installation/upgrading/#tls-ssl-changes
[5]: https://github.com/sensu/sensu-omnibus/blob/master/platform-docs/MAC_OS_X.md

> [6]: https://docs.sensu.io/sensu-go/latest/operations/maintain-sensu/migrate/

Author: ```suggestion
[6]: https://docs.sensu.io/sensu-go/latest/operations/maintain-sensu/migrate/
[7]: https://blog.sensu.io/eol-schedule-for-sensu-core-and-enterprise

```
